src/main/webapp/js/chart-loader.js

```diff
@@ -0,0 +1,25 @@
+ google.charts.load('current', {packages: ['corechart']});
+    google.charts.setOnLoadCallback(drawChart);
```

Comment:
Please pay attention to indentation when writing new code (when modifying existing code, I know that sometimes you need to also be careful of not modifying existing indentation too much - but this is new code). In particular, this line, the next, and the last line shouldn't have be indented at all.